apitools/base/py/credentials_lib.py

```diff
@@ -13,7 +14,9 @@
 import oauth2client.locked_file
 import oauth2client.multistore_file
 import oauth2client.service_account
-import oauth2client.tools  # for flag declarations
+# The `tools` import is rarely needed, but takes a long time.
```

Comment:
It's still [needed][1] for the `gflags` declarations because it imports `old_run`. The "long time" thing is what I was getting hung up on.

[1]: https://github.com/google/oauth2client/blob/0a6241c792f0c833f2f176292b0c7ea5623eabfd/oauth2client/old_run.py#L36-L47